contentful/entry.py

```diff
@@ -57,11 +57,11 @@ def _coerce(self, field_id, value, localized, includes):
         )
 
     def _build_nested_resource(self, value, localized, includes):
-        # Maximum include Depth is 10 in the API, but we raise it to 20,
+        # Maximum include Depth is 10 in the API, but we raise it to 20 (default),
```

Comment:
Because when you have say a reference on the 10th level that has a back reference to a previous level, we want to resolve (by default) the include tree for that entry as well.